Eatery/API.swift

```diff
@@ -0,0 +1,740 @@
+//  This file was automatically generated and should not be edited.
+
+import Apollo
+
+public final class AllEateriesQuery: GraphQLQuery {
+  public let operationDefinition =
+    "query AllEateries {\n  eateries {\n    __typename\n    id\n    name\n    nameShort\n    slug\n    eateryType\n    about\n    phone\n    location\n    campusArea {\n      __typename\n      descriptionShort\n    }\n    paymentMethods {\n      __typename\n      swipes\n      brbs\n      cash\n      credit\n      cornellCard\n      mobile\n    }\n    coordinates {\n      __typename\n      latitude\n      longitude\n    }\n    operatingHours {\n      __typename\n      date\n      events {\n        __typename\n        startTime\n        endTime\n        description\n        calSummary\n        menu {\n          __typename\n          category\n          items {\n            __typename\n            item\n            healthy\n          }\n        }\n      }\n    }\n  }\n}"
```

Comment:
Multi-line string literal would make this clearer.

Reply:
This gets auto-generated by GraphQL